templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -507,17 +560,25 @@ contents:
     # Check if there is a valid hint inside iface_default_hint_file. If so, use that hint.
     # If there is no valid hint, use the default interface that we found during the step
     # earlier. Write the default interface to the hint file.
-    get_default_interface() {
+    get_nodeip_interface() {
```

Comment:
Please update the comment above this function that documents how it works with the changes done.